sample_workloads/lit-gpt-demo/helm/templates/litgpt.yaml

```diff
@@ -145,18 +146,20 @@ spec:
         value: "{{$root.Values.network.disablePmtu}}"
       - name: CPU_PINNING_MODE
         value: "{{$root.Values.network.cpuPinningMode}}"
-      - name: GCS_BUCKET
-        value: "{{$root.Values.workload.gcsBucket}}"
+      - name: GCS_EXPERIMENT_BUCKET
+        value: "{{$root.Values.workload.gcsExperimentBucket}}"
       - name: EXPERIMENT_ROOT_DIR
         value: "{{$root.Values.workload.experimentDir}}"
+      - name: GCS_DATA_BUCKET
+        value: "{{$root.Values.workload.gcsDataBucket}}"
       - name: DATA_DIR
         value: "{{$root.Values.workload.dataDir}}"
       - name: BATCH_SIZE
         value: "{{$root.Values.workload.batchSize}}"
       - name: MICRO_BATCH_SIZE
         value: "{{$root.Values.workload.microBatchSize}}"
       - name: MODEL_NAME
-        value: "{{$root.Values.workload.modelName}}"
+        value: "pythia-6.9b"
```

Comment:
Undo this change, hardcoding model name.

Reply:
Fixed